executor/executor.go

```diff
@@ -349,16 +349,18 @@ func Run(path string, env []string, emitter screwdriver.Emitter, build screwdriv
 			code = <-eCode
 		case buildTimeout := <-invokeTimeout:
 			handleBuildTimeout(f, buildTimeout)
-
 			if firstError == nil {
 				firstError = buildTimeout
 				code = 3
 			}
+			_ = c.Process.Signal(syscall.SIGABRT)
 		case stepAbort := <-sig:
+			f.Write([]byte{4})
```

Comment:
is this needed ? will sigabrt kill the pty ?

Reply:
yes, otherwise the shell is in the background; doesn't affect anything visibly though.